<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="input_1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>协议id:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="box_1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>proto文件:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="input_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>301</width>
     <height>531</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>待解析内容:</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="out_1">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>210</y>
     <width>471</width>
     <height>531</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>170</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>输出:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_1">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>280</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>解析</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>130</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>协议_枚举:</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="out_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>130</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>包头长度:</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="out_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>正文长度:</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="out_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>560</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>解析_录制</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
